<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>267</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>916</width>
    <height>267</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>916</width>
    <height>267</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DreamTeamTrad - Patcheur automatique pour Zero Escape: The Nonary Games</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/ressource/DreamteamLogo.ico</normaloff>:/ressource/DreamteamLogo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="pushButton_browse">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_process">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777211</height>
     </size>
    </property>
    <property name="text">
     <string>Patcher</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gamePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Indiquer le chemin du dossier du jeu</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_choicePatch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Choix des patchs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QCheckBox" name="checkBox_999">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>9 Hours, 9 Doors, 9 Persons</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Launcher">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Launcher</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_VLR">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Virtue's Last Reward</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_log">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
